twarc.py

```diff
@@ -140,7 +141,16 @@ def main():
         tweets = t.filter(track=args.track, follow=args.follow,
                           locations=args.locations)
     elif args.hydrate:
-        tweets = t.hydrate(open(args.hydrate, 'rU'))
+        f_names = [args.hydrate] if args.hydrate != '--hydrate' else None
```

Comment:
The idea was to read from stdin if `-` is passed to pipe ids into twarc:

```
zcat ids.csv.gz | twarc.py --hydrate -
```

but in this case `args.hydrate` will somehow be equal to `--hydrate`.